workers/data_refinery_workers/processors/tximport.R

```diff
@@ -0,0 +1,63 @@
+# This script is based on the one written by J. Taroni:
+# https://github.com/jaclyn-taroni/ref-txome/blob/master/4-athaliana_tximport.R
+#
+# Two required arguments:
+# (1) --exp_dir    <experiement_dir> (directory where "quant.sf" files are found)
+# (2) --gene2txmap <path_of_gene2txmap_file>
+#
+# Two Output files:
+# (1) RDS file (saved as <experiment_dir>/txi_out.RDS)
+# (2) TPM file (saved as <experiment_dir>/gene_lengthScaledTPM.tsv)
+
+# Handle input arguments:
+library("optparse")
+option_list <- list(
+  make_option("--exp_dir", type = "character"),
+  make_option("--gene2txmap", type = "character")
+)
+
+opt_parser <- OptionParser(option_list = option_list)
+opt <- parse_args(opt_parser)
+
+# directory of the experiment data
+exp.directory <- opt$exp_dir
```

Comment:
I think we should keep the style consistent and have this become `exp_directory`